src/App.tsx

```diff
@@ -1,26 +1,109 @@
 /* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Error } from './components/Error/Error';
+import { Footer } from './components/Footer/Footer';
+import { Todos } from './components/Todos/Todos';
+import { AddTodos } from './components/AddToDo/AddToDo';
+import { DoAllTodosComplete } from './components/DoUnDoAll/DoUnDoAll';
+import { FakeToDo } from './types/fakeTodo';
+import { TodoItem } from './components/TodoItem/TodoItem';
+import { handleFiltering } from './utils/handleFiltering';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMesage, setErrorMesage] = useState<string>('');
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const [disabled, setDisabled] = useState<boolean>(false);
+  const [fakeTodo, setFakeTodo] = useState<FakeToDo | null>(null);
+  const [loader, setLoader] = useState<number | null>(null);
+  const [activeFilter, setActiveFilter] = useState(Status.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(response => {
+        setTimeout(() => {
+          setTodos(response);
+        }, 300);
+      })
```

Comment:
Don't need to use setTimeout here, this delay should be on request
```suggestion
      .then(setTodos)
```